Gemfile

```diff
@@ -2,3 +2,13 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in request_store.gemspec
 gemspec
+
+case Gem::Version.new(RUBY_VERSION.dup)
+when ->(ruby_version) { ruby_version >= Gem::Version.new('2.2.0') }
+  gem 'rake', '~> 13'
+when ->(ruby_version) { ruby_version >= Gem::Version.new('2.0.0') }
+  gem 'rake', '~> 12'
+else
+  gem 'rake', '~> 11'
+end
```

Comment:
Old versions of Ruby are incompatible with new versions of Rake and vice versa.

Avoid the incompatible combinations.

Reply:
Thank you! I believe this means this would fix https://github.com/steveklabnik/request_store/issues/88, is that right?

Reply:
oops that was rack and this is rake.